### PR TITLE
handle request error and call /TR/tr-outdated-spec only on www.w3.org

### PR DESCRIPTION
**Pull request by deniak**
cc @marcoscaceres

### CODE REVIEW

**Review by dontcallmedom — APPROVED** [review]

File: src/fixup.js [hunk 1]
```diff
@@ -192,5 +193,11 @@
     document.body.appendChild(node);
   };
 
-  request.send();
+  request.onerror = function() {
+    console.error("Request to https://www.w3.org/TR/tr-outdated-spec failed.");
+  };
+
+  if (document.location.hostname === "www.w3.org") {
```
Comment (marcoscaceres) [inline src/fixup.js]: I wonder if this `if` should wrap all of the above... you are setting up listeners, etc, that won't ever get fired.

Comment (deniak) [reply src/fixup.js]: I thought it wouldn't have any impact on performance but ok to move *all* the code inside the `if`

File: src/fixup.js [hunk 2]
```diff
@@ -141,56 +141,63 @@
...
+          isOpen = !isOpen;
+          node.classList.toggle("outdated-collapsed");
+          document.body.classList.toggle("outdated-spec");
+          button.innerText = (isOpen) ? '\u25BE collapse' : '\u25B4 expand';
+        }
+      }
+      document.body.appendChild(node);
+    };
+
+    request.onerror = function() {
```
Comment (marcoscaceres) [inline src/fixup.js]: This should be inside the if too... or else request will be undefined.

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
In the future, we should probably make a `function showDeprecationWarning(){}` and then `if(onw3cserver){ showDeprecationWarning()}`. That would make this a little more consistent with the rest of the code.

**Pull request merged**